pages/posts/flight-of-a-byte.tsx

```diff
@@ -0,0 +1,112 @@
+    }
+    highlight()
+  }, [])
+
+  return (
+    <>
+      <Head>
+        <title>Wesh Network</title>
+        <meta name="description" content="Wesh Network" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <main className={styles.Body}>
+        <Navbar />
+        <section className={styles.SectionBlog}>
+          <div className={styles.SectionBlogContainer}>
+            <div className={styles.SectionBlogCard}>
+              <div className={styles.SectionBlogText}>
+                <h2>Wesh: Flight of a Byte</h2>
+                <p>
+                In the previous blog post, we made an example app where the Wesh user account of <code>“client2”</code> sends a message <code className="language-go">“Hello”</code> to a Contact group which is received by the other user account of <code>“client1”</code>. You can use the example to start building your Wesh app, but for development and debugging it helps to understand more about how it works. So this blog post will be a theory dive into some of the details of Wesh communication.
+                </p>
+                <p>
+                When the Wesh app on the computer for <code>client2</code> sends the bytes of a message, what is the <code className="language-go">“flight path”</code> of the bytes of <code className="language-go">“Hello”</code> as they travel to the computer of <code>client1</code>? It&apos;s different from “traditional” networking where, for example, a computer sends the message in a UDP packet to the IP address of the other computer, as described in this Wikipedia article. Instead of a specific IP address, Wesh communication happens between libp2p peers, where the client&apos;s PeerID is the value that we&apos;ve seen in previous examples and is the same no matter how the client is connected to the network.
+                </p>
+                <h3>Peer discovery</h3>
+                <p>
+                The “glue” between a libp2p peers and a more traditional network address is <a href="https://docs.libp2p.io/concepts/discovery-routing/overview/">peer discovery</a>. There are a lot of details that we won&apos;t go into regarding types of discovery, peer authentication, etc. In this blog post, we just describe the minimal steps for how <code>client1</code> receives the message from <code>client2</code>. When the libp2p service of client1 discovers the computer of <code>client2</code>, it opens a network connection (<a href="https://www.notion.so/Backlogs-for-1-Berty-Messenger-Launch-and-2-Wesh-Network-d420a66eea244942878b31d3fc6eab4a?pvs=21">code</a>) and reads from it using a traditional byte stream (<a href="https://github.com/libp2p/go-libp2p-pubsub/blob/6d73cd4b56bf9d7fdd0c0caaed629a3d51027a75/comm.go#L66">code</a>). (We give links to code for curious readers, but this isn&apos;t a code tutorial and you can skip the code links.)
+
```

Comment:
In "network connection (code)", I put the wrong link in the Notion page. Sorry. (It shouldn't be a link to our notion site.) Please change the link to:
https://github.com/libp2p/go-libp2p-pubsub/blob/6d73cd4b56bf9d7fdd0c0caaed629a3d51027a75/comm.go#L115